Server/src/routes/getDatasetRouter.ts

```diff
@@ -0,0 +1,27 @@
+import { Request, Response, Router } from 'express';
+import { getDataController } from '../controllers/getDatasetController';
+
+/**
+ * This file contains the route for a call to query or obtain one or more data sets. 
+ * If an API call is made to /dataset* then the request is routed to the getDataController
+ * to continue processing of the request.
+ */
+
+let router = Router();
+
+router.get('/dataset/uploadedBy/:uploadedBy', (request: Request, response: Response) => {
+    let getDataControllerObject = new getDataController();

```

Comment:
Maybe a better way could be to create the new object like router and only have to create 1 obj. Then call the respective methods under the router

Reply:
Could you explain this another way? I have a hard time understanding what you would like done. 👍

Reply:
Good catch, will do that!